content/topics/redux-thunks/_index.md

```diff
@@ -6,7 +6,17 @@ prerequisites:
   - topics/redux-intro
   soft: []
 title: Redux Thunks
+ready: True
 ---
 
+Redux Thunk is a really confusing thing when you first hear about it.especailly that word “thunk.” So let’s clear that up first.
+
+Redux Thunk middleware allows you to write action creators that return a function instead of an action. The thunk can be used to delay the dispatch of an action, or to dispatch only if a certain condition is met,so instead of performing some work now, you produce a function body or unevaluated expression (the “thunk”) which can optionally be used to perform the work later.
+
+## Why are Redux-Thunks useful?
+
+Without middleware, Redux store only supports synchronous data flow.Thunks extend the store's abilities, and lets you write async logic that interacts with the store.
+
+For more details on thunks, see:
```

Comment:
This makes it sound like it's optional. Rather let's put in a firm instruction.

The students really should read through the first link. Rather say:

"The official Redux tutorial explains thunks really well. Please read the following and make sure you understand it"